airflow/providers/google/common/hooks/base_google.py

```diff
@@ -674,7 +674,10 @@ async def acquire_access_token(self, timeout: int = 10) -> None:
 
         self.access_token = cast(str, self.credentials.token)
         self.access_token_duration = 3600
-        self.access_token_acquired_at = datetime.datetime.now(tz=datetime.timezone.utc)
+        # access_token_acquired_at is specific to gcloud-aio's Token. On subsequent calls of `get` it will be used
+        # with `datetime.datetime.utcnow()`. Therefore we have to use an offset-naive datetime.
+        # https://github.com/talkiq/gcloud-aio/blob/f1132b005ba35d8059229a9ca88b90f31f77456d/auth/gcloud/aio/auth/token.py#L204
```

Comment:
Just want to mention, that in 5.2 the type is changed to the aware:
https://github.com/talkiq/gcloud-aio/blob/4979115547cf07bc93878158e86263833e3839fc/auth/gcloud/aio/auth/token.py#L220

It is worthwhile to add this info into the `provider.yaml` for avoid surprised when someone want to relax requirements

https://github.com/apache/airflow/blob/5a0be392e66f8e5426ba3478621115e92fcf245b/airflow/providers/google/provider.yaml#L92

Reply:
Nice! In terms of avoiding surprises, I've adjusted the test of `get` such that it certainly will make sure that `access_token_acquired_at` is compatible.

Reply:
When upgrading to gcloud-aio-auth 5 we will override the [refresh](https://github.com/talkiq/gcloud-aio/blob/4979115547cf07bc93878158e86263833e3839fc/auth/gcloud/aio/auth/token.py#L230) method and thus don't have to deal with `access_token_acquired_at` and other boilerplate code anymore.